bin/app.ts

```diff
@@ -41,6 +41,11 @@ export class RoutingAPIStage extends Stage {
       provisionedConcurrency: number;
       ethGasStationInfoUrl: string;
       chatbotSNSArn?: string;
+      stage: string;
+      route53Arn?: string;
+      pinata_key: string;
+      pinata_secret: string;
+      hosted_zone: string;
```

Comment:
Can we make these optional, so that we can deploy the API locally without needing to specify these?